test/system/index_action_partials_test.rb

```diff
@@ -0,0 +1,55 @@
+require 'application_system_test_case'
+
+class IndexActionPartialsTest < ApplicationSystemTestCase
+  def setup
+    @user = create :user
+    @client = create :client
+    login_as @user
+  end
+
+  test 'client list shows all actions needed' do
+    visit clients_path
+    assert page.has_link? 'Show', href: client_path(@client)
+    assert page.has_link? 'Edit', href: edit_client_path(@client)
+    assert page.has_link? 'Print', href: client_path(@client) + '?print=true'
+    assert page.has_link? 'Journal', href: client_journals_path(@client)
+    assert page.has_link? 'Delete', href: client_path(@client)
+  end
+
+  test 'click on print link leads to new page that prints' do
+    visit clients_path
+    assert_equal page.windows.count, 1
+    click_link 'Print'
+    assert_equal page.windows.count, 2
+    page.switch_to_window(page.windows[1])
+    assert page.has_link? @client.user.email
+  end
+
+  test 'volunteer list shows all actions needed' do
+    volunteer = create :volunteer
```

Comment:
It makes sense to keep it here, otherwise the volunteer would be created for all other tests as well.

But I can recommend adding empty lines between the setup/action/expectation stages in a test, it makes them more readable.